core/tests/Test.Microsoft.Identity.LabInfrastructure/LabServiceApi.cs

```diff
@@ -87,7 +97,7 @@ private LabResponse GetLabResponseFromAPI(UserQueryParameters query)
                 throw new LabUserNotFoundException(query, "No lab user with specified parameters exists");
             }
 
-            LabResponse response = JsonConvert.DeserializeObject<LabResponse>(result);
+LabResponse response = JsonConvert.DeserializeObject<LabResponse>(result);
```

Comment:
nit: indentation

Reply:
done.

---
In reply to: [234387018](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1405#discussion_r234387018) [](ancestors = 234387018)